config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```
@@ -462,7 +462,7 @@ postsubmits:
         - images/kubekins-e2e/
   - name: post-test-infra-push-krte
     cluster: test-infra-trusted
-    run_if_changed: '^images/krte/'
+    run_if_changed: '^(images/)(krte/|kubekins-e2e/variants.yaml)'
```

The first capture group here is unnecessary, the second part is incorrectly split into two groups:
```suggestion
    run_if_changed: '^images/(krte/)|(kubekins-e2e/variants.yaml)'
```

---

@BenTheElder I believe the proposed change here would incorrectly match `kubekins-e2e/variants.yaml` with any preceding path. The first capture group is unnecessary, but it looks like your change is splitting the second part into two capture groups?

---

I removed first capture group and added trivial change in `images/krte/`

---

Ack